src/message.rs

```diff
@@ -796,6 +859,21 @@ impl Message {
         }
     }
 
+    /// If message is a Pre-Message,
+    /// then this returns the viewtype it will have when it is downloaded.
+    pub fn get_post_message_viewtype(&self) -> Option<Viewtype> {
```

Comment:
this function exist only to be exposed to UI, and is not needed by core.

it was discussed, that we do not want UI changes atm.

therefore, the function should not exist. 

apart from that, the things is also subject to change, we probably do not want to have smth as get_post_message_viewtype() in the long run: 

- for now, message type will change, or whole message being exchanges.

- in the future, when we do the UI changes, the message type should already be set on premessage -there is not much point in UI using different getters depending on download state. depending on the download state UI will probably render some stub instead.